Lib/pathlib.py

```diff
@@ -680,22 +680,26 @@ def is_reserved(self):
         name = self._tail[-1].partition('.')[0].partition(':')[0].rstrip(' ')
         return name.upper() in _WIN_RESERVED_NAMES
 
-    def match(self, path_pattern):
+    def match(self, path_pattern, case_sensitive=None):
```

Comment:
```suggestion
    def match(self, path_pattern, *, case_sensitive=None):
```

Keyword-only to match `glob()`.